src/components/chip-set/chip-set.scss

```diff
@@ -45,12 +50,6 @@ $leading-icon-space: 1.5rem;
     min-height: shared_input-select-picker.$height-of-mdc-text-field;
     position: relative;
 
-    &.chip-set--with-label {
-        .lime-floating-label--float-above {
```

Comment:
before:

https://github.com/user-attachments/assets/27f71da5-f3b6-4f93-8c23-b4395b9068eb

after:

https://github.com/user-attachments/assets/9de20001-19f4-40ae-9cc5-26c0a147e795

Reply:
🤗 I love your review @LucyChyzhova ! Thanks for checking all these details and finding these issues. 
I'm gonna add fixups

Reply:
https://github.com/user-attachments/assets/48513da3-fcf0-4a45-b879-89fa759e274c

on my machine it clears the thing they have typed. 🤔 which I kind of like
maybe we should leave it as is :😆

Reply:
it's because you have a check for `Empty input on blur`

Reply:
i'm not sure if you decided to fix it or not 😀

Reply:
⚡ yes, fixed and pushed ;)